.jscodeshift.json

```diff
@@ -1,11 +1,18 @@
 {
   "globals": {
     "os.alertManager": {"require": "os.alert.AlertManager", "singleton": true},
+    "os.areaManager": {"require": "os.AreaManager", "singleton": true},
+    "os.ui.areaManager": {"require": "os.AreaManager", "singleton": true},
```

Comment:
Are these requires correct? I would think they should be `os.query.AreaManager`. Similar question with `FilterManager` below.

Reply:
Huh - not sure why I did that, but those are wrong. Fixed!